### ISSUE TITLE
Buffer overflow in Main.c

### ISSUE THREAD

**Issue by ProDigySML** [mach-kernel/cadius#12]
A buffer overflow was detected in Main.c. The local_buffer variable takes the value of argv[4] without any bounds checking which causes the issue. 

The assignment of the user-controlled variable is given below:
https://github.com/mach-kernel/cadius/blob/a9dd04a61f32ab220ee2f710bc9b7777a52e2579/Src/Main.c#L897

A screenshot of the crash is given below:
![image](https://user-images.githubusercontent.com/16996819/37033996-f3523fba-219b-11e8-8c3b-f6b793d8e746.png)

**Comment by mach-kernel** [comment]
Hi @ProDigySML, thanks for the bug report. I'll plug this into my next batch of fixes. I must say that I am surprised to see someone using this tool in this manner. 

Thanks!

**Comment by mach-kernel** [comment]
```
$ ./cadius CREATEVOLUME abc abc `python2 -c 'print "A" * 296 + "BBBBBB"'`                                                                                                             2.4.0
./cadius v 1.3.1, (c) Brutal Deluxe 2011-2013.
  Error: Argument too long!
```

This is fixed and will ship with #13. I feel that this is kind of contrived, but at the very least it may prevent someone from potentially messing up their local disk images. :+1:
